x/concentrated-liquidity/msg_server_test.go

```diff
@@ -138,51 +139,61 @@ func (suite *KeeperTestSuite) TestCollectFees_Events() {
 			expectedCollectFeesEvent:      3,
 			expectedMessageEvents:         4, // 1 for collect fees, 3 for send messages
 		},
+		"error: not owner with three position IDs": {
+			upperTick:                  DefaultUpperTick,
+			lowerTick:                  DefaultLowerTick,
+			positionIds:                []uint64{DefaultPositionId, DefaultPositionId + 1, DefaultPositionId + 2},
+			shouldSetupUnownedPosition: true,
+			numPositionsToCreate:       2,
+			expectedError:              cltypes.NotPositionOwnerError{},
+		},
 		"error": {
 			upperTick:                     DefaultUpperTick,
 			lowerTick:                     DefaultLowerTick,
 			positionIds:                   []uint64{DefaultPositionId, DefaultPositionId + 1, DefaultPositionId + 2},
 			numPositionsToCreate:          2,
 			expectedTotalCollectFeesEvent: 0,
-			expectedCollectFeesEvent:      0,
-			expectedMessageEvents:         2, // 2 emitted for send messages
-			expectedError:                 cltypes.PositionIdNotFoundError{PositionId: DefaultPositionId + 2},
+			expectedError:                 cltypes.NotPositionOwnerError{Address: suite.TestAccs[0].String(), PositionId: DefaultPositionId + 2},
 		},
 	}
 
 	for name, tc := range testcases {
 		suite.Run(name, func() {
 			suite.SetupTest()
-			ctx := suite.Ctx
+
+			msgServer := cl.NewMsgServerImpl(suite.App.ConcentratedLiquidityKeeper)
```

Comment:
Out of curiosity, what was the reasoning behind moving this line above the pool/position setup?

Reply:
No reason. This can be moved back if desired.

I think I might have suspected that ordering was causing some of the initial issues I was debugging and didn't eventually revert it.